python/lsst/obs/base/instrument.py

```diff
@@ -42,6 +53,22 @@ class Instrument(metaclass=ABCMeta):
     each of the Tasks that requires special configuration.
     """
 
+    policyName = None
+    """Instrument specific name to use when locating a policy or configuration
```

Comment:
I can't lower case it unless we change obs_lsst which uses lsstCam as the policy/config name.

I don't understand why getName exists. `LsstCam.name` and `HyperSuprimeCam.name` seems better anyway.

Reply:
The more I think about it, the more I think your `policyName` should just be `Instrument.name`. That keeps everything consistent, and means we don't have to define this particular thing. Changing lsstCam in that obs data package shouldn't affect anything else, since this is the only place that knows about it.

Please file a ticket to make `.name` a readonly `@property` and get rid of `getName()`; I too am confused as to why the latter exists.

Reply:
Switching policy/config to use DECam rather than decam is a big change that will affect lots of people. I think big enough that it would need an RFC. I certainly can not do that on this ticket (there are many config files that call other config files with an assumed path hierarchy).

A readonly class property is not as easy as it sounds because `@property` doesn't work. I see astropy has a `@classproperty` that might do the job.

Reply:
Good point on the `@property`. Lets just get rid of `getName` then; `.name` being not readonly is fine.

Reply:
I didn't know that the `policyName` was also used to find config files. Where does that happen, since I don't see it in your obs_decam change or here?

I want Instrument to be as simple as possible, and requiring the definition of something that we should be able to completely standardize from `.name` shouldn't be necessary.

Reply:
in obs packages that have multiple instruments (obs_subaru and obs_lsst) a form of the instrument name is used to find the instrument-specific configs. Unfortunately it's mostly lower case except for lsstCam where it's mixed case.

Reply:
I see now that you just made that change to `self.configPaths` in obs_subaru, to use `self.policyName`. Are we really planning to enshrine that the config path and the obs_data subdirectory are the same? If so, are we going to document that, and also document that those are in fact not derivable from `instrument.name`? I'm afraid our list of unenforced "suggestions" in gen3 is rapidly growing.

Reply:
The obs_x_data subdirectory was named the same as the config and policy subdirectories in an attempt to be consistent with existing usage. It's really obs_lsst:lsstCam that rocked the boat a little by using mixed case.

Reply:
Changing this will need an RFC.